pkg/installmanager/installmanager.go

```diff
@@ -1260,9 +1260,17 @@ func updateClusterProvisionWithRetries(provision *hivev1.ClusterProvision, m *In
 }
 
 func cleanupLogOutput(fullLog string) string {
-	var cleanedString string
-
-	cleanedString = multiLineRedactLinesWithPassword.ReplaceAllString(fullLog, "REDACTED LINE OF OUTPUT")
+	var cleanedString = fullLog
+
+	// The console log may have carriage returns as well as newlines,
+	// and they may be escaped (this especially happens with the
+	// baremetal IPI installer when libvirt.so emits errors). Unescape
+	// the returns and newlines, then remove the carriage returns, so
+	// that the string is treated as multiple lines.
+	cleanedString = strings.ReplaceAll(cleanedString, "\\r", "\r")
+	cleanedString = strings.ReplaceAll(cleanedString, "\\n", "\n")
+	cleanedString = strings.ReplaceAll(cleanedString, "\r\n", "\n")
```

Comment:
Is this necessary?

https://play.golang.org/p/94pQ3RKqKa-

Reply:
Not strictly, I can remove this.

Reply:
Done